src/Microsoft.Sbom.Api/Executors/LicenseInformationFetcher.cs

```diff
@@ -101,9 +106,18 @@ public async Task<List<string>> FetchLicenseInformationAsync(List<string> listOf
                     extractedLicenses.TryAdd($"{packageName}@{packageVersion}", declaredLicense);
                 }
             }
+
+            // Filter out undefined licenses.
+            foreach (var kvp in extractedLicenses.Where(kvp => kvp.Value == "NOASSERTION" || kvp.Value == "Unlicense" || kvp.Value == "OTHER").ToList())
```

Comment:
Do we need this check to be case-insensitive?

Reply:
Not necessarily but I've added it to be safe.